internal/distro/distro_test.go

```diff
@@ -42,7 +42,8 @@ func TestDistro_Pipeline(t *testing.T) {
 			continue
 		}
 		t.Run(tt.Compose.OutputFormat, func(t *testing.T) {
-			d := distro.New(tt.Compose.Distro)
+			distros := distro.NewDistroRegister([]string{"../.."})
+			d := distros.GetDistro(tt.Compose.Distro)
```

Comment:
We're now creating the distro objects every time they're being accessed. I think that's idempotent right now, so we are fine, but we might want to move the register object into something that's passed around.

Reply:
Agreed, for now this tries to minimize the changes, but should be followed by a clean-up.